src/analysis/convex/cone.lean

```diff
@@ -45,9 +49,6 @@ While `convex` is a predicate on sets, `convex_cone` is a bundled convex cone.
 
 * https://en.wikipedia.org/wiki/Convex_cone
 
-## TODO
-
-* Define the dual cone.
```

Comment:
This should probably now be mentioned elsewhere in the docstring

Reply:
I've already added "We define `set.inner_dual_cone` to be ...". Would you like to formulate this differently?

Reply:
I somehow missed this